backend/controllers/probationDocuments.js

```diff
@@ -0,0 +1,138 @@
+const moment = require('moment')
+const { properCaseName } = require('../utils')
+const config = require('../config')
+const { logError } = require('../logError')
+const { formatTimestampToDate } = require('../utils')
+
+const serviceUnavailableMessage = 'Sorry, the service is unavailable'
+const offenderNotFoundInProbationMessage =
+  'We are unable to display documents for this prisoner because we cannot find the offender record in the probation system'
+const getOffenderUrl = offenderNo => `${config.app.notmEndpointUrl}offenders/${offenderNo}`
+
+const probationDocumentsFactory = (oauthApi, elite2Api, communityApi, systemOauthClient) => {
+  const renderTemplate = (req, res, pageData) => {
+    const { pageErrors, offenderDetails, ...rest } = pageData
+
+    res.render('probationDocuments.njk', {
+      title: 'Probation documents - Digital Prison Services',
+      errors: [...req.flash('errors'), ...pageErrors],
+      offenderDetails,
+      ...rest,
+    })
+  }
+
+  const displayProbationDocumentsPage = async (req, res) => {
+    const pageErrors = []
+
+    const ensureAllowedPageAccess = userRoles => {
+      if (!userRoles.find(role => role.roleCode === 'VIEW_PROBATION_DOCUMENTS')) {
+        throw new Error('You do not have the correct role to access this page')
+      }
+    }
+
+    const getCommunityDocuments = async offenderNo => {
+      const sentenceLength = sentence => {
+        if (typeof sentence.originalLength === 'undefined' || typeof sentence.originalLengthUnits === 'undefined') {
```

Comment:
Would just checking for falsely work here? `if(!sentence.originalLength || !sentence.originalLengthUnits)` 

Just checking that you only want to display the sentence length if both length and units are present?